### DISCUSSION TITLE
小麥注音 2.7.2 版

### DISCUSSION THREAD

**Discussion by lukhnos** [openvanilla/McBopomofo#469] [Announcements]
這一版修正了以下問題：

- 版本更新通知對話盒的按鈕位置錯誤

其他的主要更動還包括了日期巨集、聯想詞、點字支援、進階輸出選項、數字轉換、線上字典查找等改進。詳情請參閱 [2.7 版發布說明](https://github.com/openvanilla/McBopomofo/releases/tag/2.7) 以及 [2.7.1 版發布說明](https://github.com/openvanilla/McBopomofo/releases/tag/2.7.1)。

下載連結: https://github.com/openvanilla/McBopomofo/releases/download/2.7.2/McBopomofo-Installer-2.7.2.zip

安裝包 MD5 hash: `a254ed437f6e21752bdd15e2fb9f5c8b`

**Comment by babushika** [comment]
目前所使用的是 McBopomofo v2.7.2 (1595) + MacOS 14.7

我已將字詞加入了用戶字典，但是仍舊會需要自己再以方向鍵叫出選擇欄後，才能選到想要的詞，而不會自動先跳出，請問這樣是的情況是正確的嗎？謝謝。

比方說打站姿二字：

https://github.com/user-attachments/assets/8e0a3785-be41-481f-b1dd-5b79f8db2f0b

**Comment by zonble** [comment]
您好：

由於您提供的資訊有點少，所以還不能夠明確知道具體的狀況。

小麥注音的演算法中，大致上是透過判斷同一組注音當中，優先選擇分數最高的組合。不過，我們有多個不同的變數，會影響一組字/詞的變數，比方說，如果最近曾經手動選擇過一組詞彙，就會動態加高這一組詞彙的分數，以影片中的範例來說，雖然有「站姿」這個詞彙，但如果曾經手動將「姿」改成選「資」，那麼「站資」的分數的確有可能就比「站姿」高。

所以這邊就需要提供一下詳細的操作

1. 前面提到將詞加到了用戶字典中，不過，內建詞庫裡頭應該已經有了「站姿」，可能要確定一下是不是確實加入了詞庫。
2. 是否曾經有過將「姿」改成「資」這樣的操作。

謝謝！


On Mon, Oct 14, 2024 at 3:16 PM babushika ***@***.***> wrote:

> 目前所使用的是 McBopomofo v2.7.2 (1595) + MacOS 14.7
>
> 我已將字詞加入了用戶字典，但是仍舊會需要自己再以方向鍵叫出選擇欄後，才能選到想要的詞，而不會自動先跳出，請問這樣是的情況是正確的嗎？謝謝。
>
> 比方說打站姿二字：
>
>
> https://github.com/user-attachments/assets/8e0a3785-be41-481f-b1dd-5b79f8db2f0b
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/openvanilla/McBopomofo/discussions/469#discussioncomment-10933009>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAM6E72I4TJ5QPFYVWHVHTZ3NVTDAVCNFSM6AAAAABP4KX2CSVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAOJTGMYDAOI>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***
> com>
>


-- 
*Weizhong Yang (a.k.a zonble)*
http://zonble.net
***@***.***
***@***.***
